### ISSUE TITLE
RCS1229 is not reported when using try/catch/finally

### ISSUE THREAD

**Issue by atifaziz** [JosefPihrt/Roslynator#753]
**Product and Version Used**: 3.0.0

[RCS1229](https://github.com/JosefPihrt/Roslynator/blob/master/docs/analyzers/RCS1229.md) is not reported when using `try` with `catch` and/or `finally`, especially when you consider that `using` is just `try`-`finally` in disguise. For example:

```cs
async Task<int> DoSomething()
{
    var service = CreateService();
    try
    {
        return service.GetAsync(); // Should report RCS1229 because of missing await keyword
    }
    finally
    {
        service.Dispose();
    }
}
```

---
/cc @p-schuler
